views/about.erb

```diff
@@ -0,0 +1,43 @@
+<html>
+    <head>
+      <title>Learn a little about me</title>
+      <meta charset = "utf-8">
+      <meta name="description" content="This page tells a little about the author of this website, Emily Gordon">
+      <meta property="og:image" content="/emilycarrot.jpg">
+    </head>
+
+    <body>
+      <header>
+        <h1>About Me</h1>
+        <img src = "./emilycarrot.jpg" alt = "Emily"/>
+      </header>
+
+      <article>
+        <p>
+          Welcome! Since you are visiting my website I thought you might
+          like to know a little bit about me.
+        </p>
+        <p>
+          My name is Emily Gordon and I am currently a student at <a href = "http://www.adadevelopersacademy.org">Ada Developers Academy</a>.
+          Right now we are learning about HTML, and this is my first attempt at a website.
+        </p>
+        <p>
+          I grew up in the Boston area, and currently live in Seattle, WA. I love to garden, and currently
+        have a <a href = "http://www.seattle.gov/neighborhoods/p-patch-community-gardening/about-the-p-patch-program">P-patch</a> community garden up at North Seattle Community College. Here are some of my favorite things to grow:
+          <div style="color:#29A329">
+            <ul>
+              <li>Tomato</li>
+              <li>Kale</li>
+              <li>Squash</li>
+            </ul>
+          </div>
+        I also love to cook and eat delicious food with friends, and explore new places in the beautiful Pacific Northwest.
+      </p>
```

Comment:
It looks like this indentation is slightly off from the opening p tag on line 24